lib/iris/tests/integration/test_netcdf.py

```diff
@@ -210,5 +210,28 @@ def test_lazy_mask_preserve_fill_value(self):
             self.assertCDL(other_filename)
 
 
+class TestCellMeasures(tests.IrisTest):
+    def setUp(self):
+        self.fname = tests.get_data_path(('NetCDF', 'ORCA2', 'votemper.nc'))
+
+    def test_load_raw(self):
```

Comment:
I included this very deliberately.  I tripped over problems with merge and concatenate not preserving cell_measures

load_raw returned the cell measures, load_cube did not

so, i'd like to keep both of these to ensure that behaviour is maintained